src/pages/resources/reference/push-notifications/templates/carousel.md

```diff
@@ -0,0 +1,27 @@
+---
+title: AEP SDK carousel push template
+description: Carousel push template supported by Adobe Campaign Classic mobile extension.
+keywords:
+- Adobe Campaign Classic
+- Push
+- Push Template
+- Carousel
+---
+
+import Tabs from './tabs/carousel.md'
+
+# Push template - carousel
+
+A notification that shows a series of three (3) to five (5) images that scroll on/off the sides of the notification. <br />If operating in `automatic` mode, images will rotate every five (5) seconds.  If operating in manual mode, the user is required to push a button to advance forwards or backwards through the carousel.
```

Comment:
Same comment here about the br tag.

Also - for number less than 20, we shouldn't be using the numerical form for it, so I'd just make it "... of three to five images..." and so on for the other numbers.